runtime/doc/lua-bit.txt

```diff
@@ -379,5 +379,4 @@ COPYRIGHT
 Lua BitOp is Copyright (C) 2008-2012 Mike Pall.
 Lua BitOp is free software, released under the MIT license.
 
-==============================================================================
  vim:tw=78:ts=4:sw=4:sts=4:et:ft=help:norl:
```

Comment:
No objection, but eventually the parser should know how to skip this case. It shows up in plugin docs a lot, which affects e.g. `gO`.

Reply:
That's hard given the current (simple) state of the grammar. I really think we should stop trying to add kludges for every combination of vimscript quirks. If you want your docs to show up correctly in Neovim, you should stick to the strict spec (which we'd need to write first, of course).